app/models/user.rb

```diff
@@ -9,8 +9,10 @@ class User < ApplicationRecord
   has_many :favourite_movies, through: :favourites, source: :movie
   has_one_attached :avatar
 
+  enum role: [ :user, :admin ]
+  
   scope :by_name, -> { order(:name)}
-  scope :not_admins, -> { by_name.where(admin: false)}
+  scope :not_admins, -> { by_name.where("role <> ?", User.roles[:user])}
```

Comment:
You can write this like so:
`scope :not_admins, -> { whene.not(role: :admim)}`

And then you can chain scope methods where you want `User.not_admins.by_name`